src/sql/src/plan/statement/dml.rs

```diff
@@ -392,9 +393,14 @@ impl TryFrom<ExplainPlanOptionExtracted> for ExplainConfig {
             subtree_size: v.subtree_size,
             timing: v.timing,
             types: v.types,
-            reoptimize_imported_views: v.reoptimize_imported_views,
-            enable_eager_delta_joins: v.enable_eager_delta_joins,
-            enable_new_outer_join_lowering: v.enable_new_outer_join_lowering,
+            features: OptimizerFeatureOverrides {
+                enable_consolidate_after_union_negate: None,
+                enable_eager_delta_joins: v.enable_eager_delta_joins,
+                enable_new_outer_join_lowering: v.enable_new_outer_join_lowering,
+                enable_reduce_mfp_fusion: None,
+                persist_fast_path_limit: None,
+                reoptimize_imported_views: v.reoptimize_imported_views,
+            },
```

Comment:
I think you can avoid explicitly enumerating the missing fields by replacing them with a single `..Default::default()`.